src/applications/model/gaming-streaming-server.cc

```diff
@@ -128,6 +102,60 @@ GamingStreamingServer::EraseStatistics (void)
   m_totSentPackets = 0;
 }
 
+void
+GamingStreamingServer::AddNewTraffic (Ptr<RandomVariableStream> packetSize,
+                                      Ptr<RandomVariableStream> interArrivalTime)
+{
+  NS_LOG_FUNCTION (this);
```

Comment:
`<< packetSize << interArrivalTime`
pointer can be written as info, do it everywhere